### ISSUE TITLE
PSR12.Traits.UseDeclaration doesnt apply the correct indent during auto fixing

### ISSUE THREAD

**Issue by dereuromark** [squizlabs/PHP_CodeSniffer#2624]
Looks like it incorrectly counts the needed whitespace indentation around use statements now.

```php
    use TransactionTrait;

    use PermissionAwareTrait;
```
is wrongly fixed to
```php
    use TransactionTrait;
        use PermissionAwareTrait;
```

Note that the extra newline didnt have any noise whitespace, it was really just an empty one.

**Comment by dereuromark** [comment]
And another bug, false positive on reporting:
```
class Repeater implements RepeaterInterface
{
    /**
     * This is some text
     *
     * We need the configuration ... foo bar.
     */
    use BundleConfigResolverAwareTrait;
```
wrongly reports
```
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 27 | ERROR | The first trait import statement must be declared on
    |       | the line after the class opening brace
```

**Comment by gsherwood** [comment]
> And another bug, false positive on reporting:

Could you please report this as a separate issue so they can be tracked more easily. Thanks.

**Comment by gsherwood** [comment]
I've fixed the original indent issue but haven't looked at the second issue. If you don't get time to report it separately, I'll break it out tomorrow.

Thanks for these reports.
